src/mscorlib/src/System/String.Manipulation.cs

```diff
@@ -665,7 +752,39 @@ public static String Join<T>(String separator, IEnumerable<T> values)
             }
         }
 
-        // Joins an array of strings together as one string with a separator between each original string.
+        // Joins a string IEnumerable together as one string with a char separator between each original string.
+        //
+        [ComVisible(false)]
+        public static String Join(Char separator, IEnumerable<String> values) {
```

Comment:
oh I see @stephentoub comment on the old PR. I agree with @stephentoub that it isn't really needed unless we see some sort of other benefit.